packages/p2p/src/components/modal-manager/modals/email-link-verified-modal/email-link-verified-modal.jsx

```diff
@@ -15,31 +15,45 @@ const EmailLinkVerifiedModal = () => {
     );
 
     return (
-        <Modal has_close_icon is_open={is_modal_open} renderTitle={() => <></>} toggleModal={hideModal} width='440px'>
-            <Modal.Body className='email-verified-modal'>
-                <Icon icon='IcEmailVerificationLinkValid' size='128' />
-                <Text className='email-verified-modal--text' color='prominent' size='s' weight='bold'>
-                    <Localize i18n_default_text="We've verified your order" />
-                </Text>
-                <Text align='center' color='prominent' size='s'>
-                    <Localize
-                        i18n_default_text="Please ensure you've received {{amount}} {{local_currency}} in your account and hit Confirm to complete the transaction."
-                        values={{ amount, local_currency }}
-                    />
-                </Text>
-            </Modal.Body>
-            <Modal.Footer className='email-verified-modal--footer'>
-                <Button
-                    large
-                    primary
-                    onClick={() => {
-                        hideModal({ should_hide_all_modals: true });
-                        order_store.confirmOrder(is_buy_order_for_user);
-                    }}
-                >
-                    <Localize i18n_default_text='Confirm' />
-                </Button>
-            </Modal.Footer>
+        <Modal
+            has_close_icon={order_store.order_information}
```

Comment:
```suggestion
            has_close_icon
```

Reply:
i had to leave this in so when the modal is loading, the close icon shouldn't appear